### DISCUSSION TITLE
Is it possible to display the current page we're on/go to a specific page without needing to go through the menus?

### DISCUSSION THREAD

**Discussion by HighMans** [edrlab/thorium-reader#1951] [Ideas]
I know for EPUBs that have page numbers you can browse to a page by going through the menus or by a keyboard shortcut, however I was wondering if it's possible to show what page you're on/jumping to a page directly from the reader without having to go through any menu options? 

It would make it a lot easier to browse the ebook.

**Comment by danielweck** [comment]
Hello, assuming your publication has "page numbers" (e.g. in EPUB reflowable books that would be the authored "page list" in the navigation document, in EPUB fixed layout publications that would be the document number in the book spine instead of page breaks) ... then there's the "Where Am I" feature to ask Thorium to display information about the current reading location (including a trail section headings, cross-linked with the table of contents). The keyboard shortcut is `SHIFT CTRL i` (`SHIFT CTRL k` for a less intrusive popup notification instead of an interrupting modal dialog). As you probably know already, `SHIFT CTRL p` opens the "goto page" user interface and focuses the keyboard cursor in the text input field where any arbitrary page break marker can be entered (numbers, roman numerals, etc. ... depending on the authored EPUB "page list")

**Comment by danielweck** [comment]
* wiki documentation: https://github.com/edrlab/thorium-reader/wiki/Keyboard-shortcuts
* documentation website: https://thorium.edrlab.org/docs/400_ressources/402_keyboard-shortcuts/

**Comment by gautierchomel** [comment]
This is related to discussion #1799. 

The main difficulty for a reading system is that forcing display of page numbers will result in strange patterns as they can be in the middle of a phrase or a word.
